netbout-web/src/test/java/com/netbout/rest/bout/TkEmVerifyTest.java

```diff
@@ -123,8 +156,8 @@ public void rejectsUnneededVerification() throws Exception {
      */
     private static RqRegex request(final String code) throws Exception {
         return new RqRegex.Fake(
-            "(.*)",
-            URLEncoder.encode(TkEmVerifyTest.ENC.encrypt(code), "UTF-8")
+                PATTERN,
```

Comment:
@kujtimiihoxha statics should be prependended with class name, so here it should be TkEmVerifyTest.PATTERN